etc/schema.json

```diff
@@ -2717,11 +2714,23 @@
                                             "client_encoding": {
                                                 "type": "string"
                                             },
+                                            "database": {
+                                                "type": "string"
+                                            },
+                                            "datestyle": {
+                                                "type": "string"
+                                            },
+                                            "extra_float_digits": {
+                                                "type": "string"
+                                            },
+                                            "options": {
+                                                "type": "string"
+                                            },
                                             "replication": {
                                                 "type": "string"
                                             }
                                         },
-                                        "additionalProperties": false
+                                        "additionalProperties": true
```

Comment:
is there a way to enforce that the type of the members is always `string`?

Reply:
For parameters status in postgresql, the values are always sent as string. Should we use something else?

Reply:
(sorry, I needed more coffee)